packages/apps/graph/generated-schema.graphql

```diff
@@ -172,7 +172,7 @@ type Query {
   block(hash: String!): Block
   blocksFromHeight(chainIds: [Int!], startHeight: Int!): [Block!]!
   chainAccount(accountName: String!, chainId: String!, moduleName: String!): ChainModuleAccount
-  completedBlockHeights(chainIds: [String!], completedHeights: Boolean, heightCount: Int): [Block!]!
+  completedBlockHeights(chainIds: [Int!], completedHeights: Boolean, heightCount: Int): [Block!]!
```

Comment:
Why is this changed? In @kadena/client we reference by string as it could be different in the future